std/machines/split/split_gl.asm

```diff
@@ -70,11 +70,11 @@ machine SplitGL with
     let P_B: col = b;
     col fixed P_LT(i) { if a(i) < b(i) { 1 } else { 0 } };
     col fixed P_GT(i) { if a(i) > b(i) { 1 } else { 0 } };
+    [ bytes, BYTES_MAX, lt, gt ] in [ P_A, P_B, P_LT, P_GT ];
```

Comment:
This change makes the machine less readable in my opinion. Also in my example it was triggered by the lookup being after `was_lt`, does this also trigger it?

Reply:
Yes, I think I did the same and when trying to undo the change, I put it in the wrong place. Fixed now!